app/ConfigValidation/ValidateConfigApplication.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\App\ConfigValidation;
+
+use Symfony\Component\Console\Application;
```

Comment:
So this thing is completely independent from the Fundraising Frontent app PHP right? If so, I think it should not be in the app/ folder. How about a new sub folder in build/? Or perhaps a new folder altogether.